### ISSUE TITLE
new chat does not reflect immediately for new account

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#21036]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. from userA, create a new account
2. from userB, click on + icon and select New chat option
3. enter userA email and click on that
4. send any message
## Expected Result:
 message should be receive immediately on receiver side
## Actual Result: 
message does not receive immediately on receiver side

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.29.0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/feb304c8-94dc-468b-a8e4-63417b898a19

https://github.com/Expensify/App/assets/43996225/b2605c4b-f7c6-4474-97ba-8c3a6ec43551

**Expensify/Expensify Issue URL:**
**Issue reported by:** @gadhiyamanan 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686992223529569

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by marcochavezf** [comment]
This issue is happening because the `OpenReport` command is super slow. When I created a new account, one call returned 502 after 3 mins, the next one hasn't finished yet:

<img width="1512" alt="Screenshot 2023-06-19 at 11 50 24" src="https://github.com/Expensify/App/assets/6829422/551e2654-c44e-4da6-9ec1-802eb09b22a4">

There's already a [GH](https://github.com/Expensify/Expensify/issues/291725) to improve the `OpenReport` performance and there's one [PR](https://github.com/Expensify/Auth/pull/8114) that was merged, but I think it hasn't been deployed yet. I will check it out with deployers, since this issue is not happening locally (but difficult to compare the performance in my computer vs in the server).
